jsx/src/util/paginationParams.js

```diff
@@ -0,0 +1,66 @@
+import { debounce } from "lodash";
+import { useSearchParams } from "react-router-dom";
+
+export const usePaginationParams = () => {
+  // get offset, limit, name filter from URL
```

Comment:
This is the shared code for storing pagination info in URL (for any paginated page)